Packs/TrendMicroCAS/Integrations/TrendMicroCAS/TrendMicroCAS_test.py

```diff
@@ -0,0 +1,371 @@
+    assert results.outputs == USER_ACTION_RESULT_OUTPUT
+    assert results.outputs_key_field == 'batch_id'
+    assert results.outputs_prefix == 'TrendMicroCAS.UserActionResult'
+
+
+def test_blocked_lists_get_command(mocker):
+    """Tests blocked_lists_get_command function
+    Given
+    - raw response
+    When
+    - mock the client's http_request
+    Then
+    - convert the result to human readable table
+    - create the context
+    - validate the expected_result and the created context
+    """
+    mocker.patch.object(client, '_http_request', return_value=BLOCKED_LISTS_GET_RESULT)
+    results: CommandResults = blocked_lists_get_command(client)
+    assert results.outputs == BLOCKED_LISTS_OUTPUT
+    assert results.outputs_key_field == 'BlockedList'
+    assert results.outputs_prefix == 'TrendMicroCAS.BlockedList'
+
+
+def test_blocked_lists_update_command(mocker):
+    """Tests blocked_lists_update_command function
+    Given
+    - raw response
+    When
+    - mock the client's http_request
+    Then
+    - convert the result to human readable table
+    - create the context
+    - validate the expected_result and the created context
+    """
+    args = {
+        'action_type': 'action_type',
+        'senders': '456@gmail.com,123@gmail.com',
+        'urls': '123.com,456.com,789.com',
+        'filehashes': 'f3cdddb37f6a933d6a256bd98b4bc703a448c621'
+    }
+    mocker.patch.object(client, '_http_request', return_value=BLOCKED_LISTS_UPDATE_RESULT)
+    results: CommandResults = blocked_lists_update_command(client, args)
+    assert results.outputs == BLOCKED_LISTS_OUTPUT
+    assert results.outputs_key_field == 'BlockedList'
+    assert results.outputs_prefix == 'TrendMicroCAS.BlockedList'
+
+
+DATA_TEST_PARSE_DATE_TO_ISOFORMAT = [
+    ('08/09/10', '2010-08-09T00:00:00Z'),
+    ('08.09.10', '2010-08-09T00:00:00Z'),
+    ('08-09-10', '2010-08-09T00:00:00Z'),
+    ('9/10/11 09:45:33', '2011-09-10T09:45:33Z'),
+
+]
+@pytest.mark.parametrize('date_input, fan_result', DATA_TEST_PARSE_DATE_TO_ISOFORMAT)
+def test_parse_date_to_isoformat(date_input, fan_result):
+    """input few date formats and checks if "parse_date_to_isoformat" returns iso ˚format string(%Y-%m-%dT%H:%M:%SZ)"""
+    result = parse_date_to_isoformat(date_input, 'test')
+    assert result == fan_result
+
+
+DATA_TEST_PARSE_DATE_TO_ISOFORMAT_FREE_TEXT = [
+    ('1 day'),
+    ('3 months'),
+    ('1 week and 1 day')
+]
+@pytest.mark.parametrize('date_input', DATA_TEST_PARSE_DATE_TO_ISOFORMAT_FREE_TEXT)
+def test_parse_date_to_isoformat_on_free_text(date_input):
+    """input a free text date and checks if "parse_date_to_isoformat" returns iso format string(%Y-%m-%dT%H:%M:%SZ)"""
+    result = parse_date_to_isoformat(date_input, 'test')
+    try:
+        datetime.datetime.strptime(result, "%Y-%m-%dT%H:%M:%SZ")
+    except ValueError as err:
+        its_not_isoformat = True
+    assert its_not_isoformat
+
+
+SECURITY_EVENTS_LIST_RESULT = {
```

Comment:
Let's put all of this constant objects in a test data files (you can look at Hellow_world for reference.